llvm/test/tools/obj2yaml/XCOFF/aix.yaml

```diff
@@ -52,12 +52,30 @@
 # CHECK32-NEXT:     Type:            0x0
 # CHECK32-NEXT:     StorageClass:    C_EXT
 # CHECK32-NEXT:     NumberOfAuxEntries: 1
+# CHECK32-NEXT:     AuxEntries:
```

Comment:
just curiosity, you have   `AUX_CSECT` test in aux0symbols,yaml, why you add test case here ?

Reply:
NumberOfAuxEntries was set to 1 but non auxiliary symbol is defined in this case, which was allowed before but will be illegal after this PR. And `aix.yaml` is a basic test for general behavior of xcoff2yaml and I think it would be better to add such test instead of remove the set of `NumberOfAuxEntries: 1` here.

Reply:
thanks for explain.